src/post.js

```diff
@@ -0,0 +1,30 @@
+const apiUrl = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/BQIIMRTZbQu0evVyYzYL/scores';
+const ScoreList = document.querySelector('.score-list');
+
+const Displayscore = async () => {
+  const response = await fetch(apiUrl)
+    .then((res) => res.json())
+    .then((data) => data.result);
+  response.forEach((item) => {
+    const newitem = document.createElement('li');
+    newitem.innerHTML = `<h4>${item.user}&nbsp&nbsp&nbsp${item.score}</h4>`;
+    ScoreList.appendChild(newitem);
+  });
+
+  return response;
+};
+Displayscore();
```

Comment:
- [ ] [OPTIONAL] I see you're calling the `Displayscore()` method from the `post.js` file, while this is working it, might be confusing in the future when your application gets bigger. alternatively, you can call the `Displayscore()` method from your `index.js` which will make it a lot more obvious that you want the method to be run when starting the application.